forms/aws/lambda/reliability/lib/notifyProcessing.js

```diff
@@ -17,21 +18,38 @@ module.exports = async (submissionID, sendReceipt, formSubmission, message) => {
     : formSubmission.form.titleEn;
   // Need to get this from the submission now.. not the app.
   const submissionFormat = formSubmission.submission;
-  // Send to Notify
 
+  const fileInputPaths = extractFileInputResponses(formSubmission);
+  
+  // Send to Notify
   if ((submissionFormat !== null) & (submissionFormat.email !== "")) {
-    return await notify
-      // Send to static email address and not submission address in form
-      .sendEmail(templateID, "forms-formulaires@cds-snc.ca", {
-        personalisation: {
-          subject: messageSubject,
-          formResponse: emailBody,
-        },
-        reference: submissionID,
-      })
-      .catch((err) => {
-        throw new Error(`Sending to Notify error: ${JSON.stringify(err)}`);
+    return await retrieveFilesFromReliabilityStorage(fileInputPaths)
+      .then(async (files) => {
+
+        const attachFileParameters = fileInputPaths.reduce((acc, current, index) => {
+          return {
+            [`file${index}`]: {
+              "file": files[index],
```

Comment:
We may need to ensure the encoding is a string as base64 due to the return of s3 get object being a readable stream or blob.
Could be of use: https://stackoverflow.com/questions/36942442/how-to-get-response-from-s3-getobject-in-node-js

Reply:
According to a post I found on StackOverflow (can't find it right now) it was stated that the response coming from the AWS SDK is already a base64 string which is why I did not try to convert it as I wanted to see how it would behave once running on staging.